.circleci/config.yml

```diff
@@ -53,3 +51,45 @@ jobs:
       - store_artifacts:
           path: test-reports
           destination: test-reports
+  
+  build_frontend:
+    working_directory: ~/repo/frontend
+    docker:
+      - image: circleci/node:10.16.3
+    steps: # a collection of executable commands
+      - checkout:
+          path: ~/repo
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@latest'
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package-lock.json" }}
+      - run:
+          name: install-npm-wee
+          command: npm install
+      - save_cache:
+          key: dependency-cache-{{ checksum "package-lock.json" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: test
+          command: npm test
+      - run:
+          name: code-coverage
+          command: 'npm test-copverage'
```

Comment:
<strong>Fix</strong> line 79. Should be `npm run test-coverage`

To run npm scripts, run: `npm run <script_name>`.
Lifecycle scripts, such as `npm start` are exceptions and they can be runned without `run`